chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -132,9 +134,16 @@ impl PartialWitnessActor {
             "distribute_chunk_state_witness",
         );
 
+        let signer = match self.my_signer.get() {
+            Some(signer) => signer,
+            None => {
+                return Err(Error::NotAValidator);
```

Comment:
It would be good to either add context to the error or log an error. Otherwise it would be hard to say which of the cases where this error is returned is actually triggered.

Reply:
Added context https://github.com/near/nearcore/pull/11400/commits/2b04477ac39044f1fe1e69c828379395037393cc